cmd/multus-daemon/main.go

```diff
@@ -34,35 +34,17 @@ import (
 	srv "gopkg.in/k8snetworkplumbingwg/multus-cni.v3/pkg/server"
 	"gopkg.in/k8snetworkplumbingwg/multus-cni.v3/pkg/server/api"
 	"gopkg.in/k8snetworkplumbingwg/multus-cni.v3/pkg/server/config"
-	"gopkg.in/k8snetworkplumbingwg/multus-cni.v3/pkg/types"
 
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
-const (
-	multusPluginName                    = "multus-shim"
-	defaultCniConfigDir                 = "/etc/cni/net.d"
-	defaultMultusGlobalNamespaces       = ""
-	defaultMultusLogFile                = ""
-	defaultMultusLogMaxSize             = 100 // megabytes
-	defaultMultusLogMaxAge              = 5   // days
-	defaultMultusLogMaxBackups          = 5
-	defaultMultusLogCompress            = true
-	defaultMultusLogLevel               = ""
-	defaultMultusLogToStdErr            = false
-	defaultMultusMasterCNIFile          = ""
-	defaultMultusNamespaceIsolation     = false
-	defaultMultusReadinessIndicatorFile = ""
-	defaultSocketDir                    = "/run/multus/"
-)
-
```

Comment:
I take it these weren't used anymore? The only advantage is/was that you could see them at a glance...

Reply:
Ahhhh ha, I see it, these are just for the command line options. Got it. +1.

Reply:
Yeah, that's right. Now command line options are removed so it is removed. But default value is still in `pkg/server`.